sdk/test/test_get_next_stage.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from harambe.core import get_next_stage
+from harambe.core import get_previous_stage
+from harambe.types import Stage
+
+
+@pytest.mark.parametrize(
+    "previous_stage,expected_stage",
```

Comment:
I don't think this should be introduced to harambe. The consumer of the observer should make this choice